templates/zerver/apps.html

```diff
@@ -42,19 +42,19 @@ <h1>Zulip for <span class="platform"></span></h1>
         <h2>Apps for every platform.</h2>
 
         <div class="apps">
-            <a class="" href="/apps/mac">
+            <a class="/apps/mac">
```

Comment:
As I know, you shouldn't use `class` for anything else than referencing elements and here it's content is used in the functionality. I would use data attributes for that, like `data-href="/apps/mac"`.

Reply:
oh, brilliant idea, i am fixing this right now

Reply:
though i have to delete this commits and push some new commits

Reply:
@alexandraciobica  please reply

Reply:
Yes, I think the easiest way is to delete the commits and push new ones. Sorry for the late reply.

Reply:
@alexandraciobica Sorry I am very busy because of my exam, I will fix this soon

Reply:
The changes look good! 

I am really sorry for the late answer, I am really busy with the projects for university and I missed the message. 

@timabbott I don't know how to approve the changes, I don't see the Approve button.

Reply:
@alexandraciobica are you talking about resolve button?, it's in the bottom left corner of this conversation